.gitlab-ci.yml

```diff
@@ -34,13 +34,13 @@ dotnet-package:
     - echo "Creating nuget package"
     - echo "Packing nuspec file via arcane roundabout csproj process"
     - dotnet pack dotnet/Datadog.AzureAppServices.DotNet.csproj -p:NoBuild=true -p:NoDefaultExcludes=true -o package
-    - echo "Updating versions from v1_3_0 to v0_1_22 for testing package"
-    - sed -i 's/v1_3_0/v0_1_22/g' dotnet/content/applicationHost.xdt.dd
+    - echo "Updating versions from v1_3_1 to v0_1_23 for testing package"
+    - sed -i 's/v1_3_1/v0_1_23/g' dotnet/content/applicationHost.xdt.dd
     - sed -i 's/EXTENSION_VERSION" value="1.2.0"/EXTENSION_VERSION" value="0.1.21"/g' dotnet/content/applicationHost.xdt.dd

```

Comment:
Can you manually update this line? I'll fix the regex in another PR

Reply:
Doh, yeah, totally meant to do that and completely forgot 🤦‍♂️